apps/glusterfs/volume_entry_allocate.go

```diff
@@ -374,7 +374,6 @@ func (v *VolumeEntry) replaceBrickInVolume(db wdb.DB, executor executors.Executo
 	newBrick.Path = newBrickEntry.Info.Path
 	newBrick.Host = newBrickNodeEntry.StorageHostName()
 
-	node := oldBrickNodeEntry.ManageHostName()
```

Comment:
If the old node is down this breaks stuff and causes a bug. I could have copied the code block inside of prepForBrickReplacement that calls GetVerifiedManageHostname and call it a 2nd time here. But I felt that it was redundant and possibly more expensive than we wanted. I could roll node into replacementItems type, but I didn't think that was appropriate because its somewhat tangential to what's being replaced.

Reply:
Ah, yes, now I see! Missed the `node` going outside `prepForBrickReplacement()` and into `replaceBrickInVolume()`.